scripts/s3.py

```diff
@@ -170,30 +175,42 @@ def main():
     if len(data) == 0:
         raise ValueError(f"Data is empty for chain_id: {chain.id}")
 
-    out = "generated"
-    if os.path.isdir(out):
-        shutil.rmtree(out)
-    os.makedirs(out, exist_ok=True)
+    to_export = []
+    suffix = None
+    if export_mode == "endorsed":
+        to_export = [vault for vault in data if vault["endorsed"]]
+        suffix = "all"
+    elif export_mode == "experimental":
+        to_export = [vault for vault in data if not vault["endorsed"]]
+        suffix = "experimental"
 
-    vaults_api_path = os.path.join("v1", "chains", f"{chain.id}", "vaults")
+    if len(to_export) == 0:
+        raise EmptyS3Export(f"No data for vaults was found in generated data, aborting upload")
 
-    file_path = os.path.join(out, vaults_api_path)
-    os.makedirs(file_path, exist_ok=True)
+    file_name, s3_path = _get_export_paths(suffix)
+    _export(to_export, file_name, s3_path)
 
-    endorsed = [vault for vault in data if vault["endorsed"]]
-    experimental = [vault for vault in data if not vault["endorsed"]]
+    # Sent a metric so we can track and alert on if this was successfully generated
+    utc_now = datetime.utcnow()
+    send_metric(f"{METRIC_NAME}.success", 1, utc_now, tags=metric_tags)
```

Comment:
Should we add what kind of export_mode we're using in `metric_tags`? Likewise for the telegram message would be good to include whether endorsed or experimental is being released